src/content/zero-knowledge-proofs.md

```diff
@@ -143,6 +143,39 @@ This is where verifiable computation comes into play. When a node executes a tra
 
 [Zero-knowledge rollups](/developers/docs/scaling/zk-rollups) and [validiums](/developers/docs/scaling/validium/) are two off-chain scaling solutions that use validity proofs to provide secure scalability. These protocols execute thousands of transactions off-chain and submit proofs for verification on Ethereum. Those results can be applied immediately once the proof is verified, allowing Ethereum to process more transactions without increasing computation on the base layer.
 
+### Secure online voting {#secure-online-voting} 
+
+Blockchain-based voting schemes promise to remedy some of the problems associated with traditional online voting. For example, unlike regular databases, information stored on-chain is fully public and auditable and mostly resistant to tampering. Blockchains are also decentralized and resistant to censorship, which ensures no one is prevented from voting for any reason. 
+
+However, on-chain voting systems are still susceptible to one of the biggest problems affecting traditional voting: collusion. [Collusion](https://en.m.wikipedia.org/wiki/Collusion) is the act of coordinating to "limit open competition by deceiving, defrauding, misleading others." When participants in an on-chain vote [collude to swing results](https://vitalik.ca/general/2019/04/03/collusion.html), it ceases to be an effective mechanism for measuring public preferences for some specified outcomes.
+
+For example, imagine there are two options up for voting on a ballot (`option A` and `option B`). Alice prefers `option A`, but votes for `option B` after receiving a bribe from Bob. Here, Alice and Bob have colluded to mislead others about the popularity of `option A` to the detriment of `option B`. 
+
+Bribery and collusion limit the effectiveness of certain use-cases enabled by blockchain voting, especially [quadratic funding](https://www.radicalxchange.org/concepts/plural-funding/). Fortunately, newer solutions, such as MACI (Minimum Anti-Collusion Infrastructure), are using zero-knowledge proofs to mitigate this problem and increase the integrity of on-chain voting. 
```

Comment:
> Bribery and collusion limit the effectiveness of certain use-cases enabled by blockchain voting, especially quadratic funding

Would say bribery and collusion limit any form of voting that provides a way for the voter to prove how they voted. Same applies to non-digital voting... if the briber has a way to confirm how you've voted the bribe can be enforced.

Would suggest maybe just broadening that first sentence, and then lead into quadratic funding. Perhaps explain why/how this affects QF (ie. a briber gives funds to the user with demands to allocate to a particular project, knowing that project will get more than the bribe back from the matching pool, which can be verified because the votes are public)